MAUI/Masked-Entry/Getting-Started.md

```diff
@@ -141,7 +165,7 @@ maskedEntry.Mask = "[A-Za-z0-9._%-]+@[A-Za-z0-9]+.[A-Za-z]{2,3}";
 
 ![RegEx mask in .NET MAUI Masked Entry.](maskedentry_images/net-maui-masked-entry-regex-mask.png)
 
-## Setting the prompt character
+## Step 6: Setting the prompt character
```

Comment:
remove the steps for this topic. Set the topic directly

Reply:
Addressed